.github/workflows/test.yml

```diff
@@ -69,6 +70,22 @@ jobs:
         sed -i -e 's:~=:==:' requirements/*
         mv requirements/install.txt requirements/dev.txt
 
+    # To use libtree-sitter-bash.so installed on the system, we need to compile
+    # it manually, because ubuntu doesn't yet package (but Gentoo does)
+    - name: Install tree-sitter-bash
+      if: ${{ matrix.os == 'ubuntu-latest' && matrix.tree-sitter-bash == 'system' }}
+      run: |
+        sudo apt install build-essential wget
+        wget https://github.com/tree-sitter/tree-sitter-bash/archive/v0.19.0.tar.gz
+        tar xvzf v0.19.0.tar.gz
+        pushd tree-sitter-bash-0.19.0/src
+        objects=( parser.o scanner.o )
+        make CFLAGS="-I${PWD}" CXXFLAGS="-I${PWD}" "${objects[@]}"
+        g++ -Wl,-O1 -Wl,--as-needed -shared parser.o scanner.o -Wl,--soname=libtree-sitter-bash.so.13 -o libtree-sitter-bash.so.13
```

Comment:
This is heavily inspired by [tree-sitter-grammar.eclass](https://gitweb.gentoo.org/repo/gentoo.git/tree/eclass/tree-sitter-grammar.eclass#n67).